services/account.rst

```diff
@@ -0,0 +1,105 @@
+
+Science team member
+------
+
+If you are not working for the instrument development, you are categorized as 
+a science team member here, and usually you do not need any other account. 
+But depending on your roles and contributions to the studies of PFS science 
+cases and survey planning, you may need an account to use services at 
+pfs.ipmu.jp (https). 
+In this case, please go back and read `Technical team member`_. 
+In particular, if you plan to contribute to software development, 
+you should follow the procedure as follows:
+
+* Contact a local site manager at your institute in the PFS technical team and formalize your commitment(s) to the software development.
+* `Register to PFS JIRA <https://pfspipe.ipmu.jp/jira/secure/Signup!default.jspa>`_, if you need to participate in discussions by filing tickets, sending comments, and so on.
+* Send your GitHub account name by email to github at pfs.ipmu.jp, if you need to push to git repositories for software coding works, code reviews, and documentations. 
+
+Notes and tips for the registration process
+------
+
+* If you already have a pfs.ipmu.jp (https) account but cannot login to PFS JIRA, you should send an email to jira at pfs.ipmu.jp, instead of registration at signup page. 
+* You will receive email from GitHub but not from the PFS project for further registration procedures to GitHub organization. Registration process will not finish unless you follow these procedures. This registration to GitHub organization does not mean to subscribe activities of repositories, you need to subscribe repositories in interest by yourself if you want. 
+* PFS GitHub is open to the public, so you can read the contents and add comments except for those in the private repositories, but you cannot push to git repositories unless you follow the above procedure.
+* PFS JIRA is open to the public, so you can read the contents, but you cannot make any commitments therein unless you follow the above procedure.
+* In the PFS GitHub organization, there is one team per institute where the members are given admin privilege. So instead of sending email to github at pfs.ipmu.jp, you can also ask the members in your institute to add yourself with privilege given to repositories related to your institute. 
+
+Lost account or change password
+------
+
+If you lost your account information or forgot your password, ask to reset 
+by following procedures. 
+
+SuMIRe/PFS PBworks wiki
+  Access `password reset <https://my.pbworks.com/?p=forgot>`_ and follow an instruction shown.
+pfs.ipmu.jp and pfspipe.ipmu.jp
+  Send email to pfs at pfs.ipmu.jp from email address you registered.
+PFS JIRA
+  If you have an account under single sign on at pfs.ipmu.jp (https), 
+  send email to pfs at pfs.ipmu.jp. Otherwise access 
+  `JIRA registration page <https://pfspipe.ipmu.jp/jira/secure/ForgotLoginDetails.jspa>`_
+  and follow an instruction shown. 
+PFS Slack
```

Comment:
There seems no instruction to create (request) Slack account above.

Reply:
exactly! added (both to tech/sci).